### DISCUSSION TITLE
Notifications not working properly

### DISCUSSION THREAD

**Discussion by antoniovazquezblanco** [community/community#13678] [Other Feature Feedback, Questions, & Ideas]
Previous information: https://github.community/t/cannot-see-pending-notifications/218139/8

![](https://user-images.githubusercontent.com/304193/158629336-b99b14c5-8d65-4934-ba59-79294bda602f.png)

According to Github I have pending notifications but when I access the notifications page I am unable to see them!

**Comment by alexghi** [comment]
same here.. it redirect me to a 500 error octopus page

**Comment by icecream17** [comment]
Same for me as well
![image](https://user-images.githubusercontent.com/58114641/163439716-3f603ac6-c58d-480a-b574-40d314cf1318.png)

**Comment by antoniovazquezblanco** [comment]
Seems to have been fixed for me. Don't know what happened but seems gone now.
